### PR TITLE
CTCTRADERS-2554:update validation on destination trader eori -must start with GB or XI

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

**Comment by kenneth-rayner** [comment]
Test this please

File: app/controllers/ConsigneeEoriNumberController.scala [hunk 1]
```diff
@@ -91,7 +91,7 @@ class ConsigneeEoriNumberController @Inject()(
               },
               value =>
                 for {
-                  updatedAnswers <- Future.fromTry(request.userAnswers.set(ConsigneeEoriNumberPage, value))
+                  updatedAnswers <- Future.fromTry(request.userAnswers.set(ConsigneeEoriNumberPage, value.toUpperCase))
```
Comment (MatthewAlexanderBeck) [inline app/controllers/ConsigneeEoriNumberController.scala:94]: Is this required? Might be classed as changing the users answer, we should validate the actual field for upper case.

File: app/controllers/TraderEoriController.scala [hunk 2]
```diff
@@ -89,7 +89,7 @@ class TraderEoriController @Inject()(override val messagesApi: MessagesApi,
             },
             value =>
               for {
-                updatedAnswers <- Future.fromTry(request.userAnswers.set(TraderEoriPage, value))
+                updatedAnswers <- Future.fromTry(request.userAnswers.set(TraderEoriPage, value.toUpperCase))
```
Comment (MatthewAlexanderBeck) [inline app/controllers/TraderEoriController.scala:92]: Same her

File: app/controllers/ConsigneeEoriNumberController.scala (hunk 1 above)
Comment (kenneth-rayner) [reply app/controllers/ConsigneeEoriNumberController.scala]: Not according to Luke. I have made change as per Fred suggested. Need to accept blank spaces but remove them

File: app/controllers/TraderEoriController.scala (hunk 2 above)
Comment (kenneth-rayner) [reply app/controllers/TraderEoriController.scala]: as above

**Review by MatthewAlexanderBeck — APPROVED** [review]

**Review by connor-irvine — APPROVED** [review]

**Pull request merged**